test/integration/evaluate.test.js

```diff
@@ -41,4 +44,24 @@ describe('evaluate', () => {
       );
     })
   );
+
+  it(
+    'runs JavaScript passed as an Async Function with callback',
+    co.wrap(function*() {
+      if (browserName !== 'chrome') {
+        //They only work for chrome.
```

Comment:
maybe also firefox?  could we do some light feature-detection here?  what is it that doesn't work on some browsers?